lib/polyfill/mediakeys.js

```diff
@@ -52,12 +52,15 @@ shaka.polyfill.MediaKeys.install = function() {
     shaka.log.info('Using native EME as-is.');
   } else if (HTMLMediaElement.prototype.webkitGenerateKeyRequest) {
     shaka.log.info('Using prefixed EME v0.1b.');
+    /** @suppress {missingRequire} */
```

Comment:
This doesn't make sense to me.  We have 
goog.require('shaka.polyfill.PatchedMediaKeys.v01b') already.  Is that not working any more?